build.gradle

```diff
@@ -155,7 +155,7 @@ task wikiDocs(dependsOn: [wikiDocsUpdateList]) {
 }
 
 def fitnesseJar() {
-    fileTree(dir: 'dist/lib', include: 'fitnesse-*.jar').getSingleFile()
+    fileTree(dir: 'dist/lib', include: 'fitnesse-*standalone.jar').getSingleFile()
```

Comment:
@bhuesemann, have you faced any issues which required adding this `standalone` part of the jar filename? I think we are not pulling any other `fitnesse*.` jars but I might be wrong...

Reply:
@javornikolov I think it is not required for the jdk8 upgrade but will be needed when upgrading gradle itself or the gradle java plugin to java-library plugin (to get advanced dependency management). 

As a side note jdk8 is not enough anymore to run Eclipse / VSC IDE in current or closely upcoming IDE releases (they require JDK 11):
https://www.eclipse.org/lists/eclipse-pmc/msg03821.html
https://github.com/redhat-developer/vscode-java/wiki/JDK-Requirements

This is causing project setup headaches for me, as Eclipse/VSC + Gradle 4.4 doesn't support setting the Gradle jdk via .settings
"java.import.gradle.java.home" (this requires Gradle >= 4.7)

and gradle 4.4 is not able to run with jdk >= JDK10. 

So the gradle update is needed and very closely related. 

It's all intertwined and many changes are needed to get this project into shape. And additionally all individual/intermediate steps we now take require specific handling either in the project setup or gradle config just to eventually change them again later again when we get to the next step (upgrading the next component or library).

Reply:
@bhuesemann, I bumped up Gradle as well and merged it to master. Resolving deprecation is good idea as well, though I decided to leave it as separate change.

IDEs often can run in separate JRE/JDK than the JDK used to build projects within them.